components-ui/see-more/index.tsx

```diff
@@ -0,0 +1,51 @@
+import { useId, useState } from 'react';
```

Comment:
This component was meant for hiding `mission` long description behind a « see more » button, but I forgot React was not activated clientSide. So I removed `mission` property, but I keep this here, it can become handy when we move to Server Component.

Reply:
Didnot know of `useId` and I love it